apps/reference/docs/guides/database/extensions/pgnet.mdx

```diff
@@ -0,0 +1,207 @@
+---
+id: pgnet
+title: 'pg_net: Async Networking'
+description: 'pg_net: an async networking extension for PostgreSQL.'
+---
+
```

Comment:
Could we add a notice here that pg_net is still not stable yet? The interface will likely change to the one proposed on https://github.com/supabase/pg_net/issues/62 and functions will not require security definer.

```suggestion
:::caution

pg_net is not at 1.0(stable) yet. Expect the function signatures to change.
```